build.gradle

```diff
@@ -20,7 +20,7 @@ apply plugin: 'com.github.johnrengelman.shadow'
 //
 group            = 'io.vantiq'
 archivesBaseName = 'vantiq-sdk'
-version          = '1.0.32'
```

Comment:
We don't seem to have been using the "minor" version here to indicate an API addition, so I continued that tradition.  Let me know if that's not what we want.

Reply:
I would like to establish a new tradition -- when we add new features (without breaking the existing API), we bump the *feature* (aka minor) version and *not* the patch version.  So yes, IMO this should be `1.1.0`.

Reply:
Tradition established...